support-frontend/assets/helpers/productPrice/billingPeriods.ts

```diff
@@ -3,33 +3,55 @@ import type { ActiveRatePlanKey } from 'helpers/productCatalog';
 
 export enum BillingPeriod {
 	Annual = 'Annual',
+	OneYearGift = 'OneYearGift',
 	Monthly = 'Monthly',
 	Quarterly = 'Quarterly',
+	ThreeMonthGift = 'ThreeMonthGift',
 	OneTime = 'OneTime',
 }
 export type RecurringBillingPeriod =
 	| typeof BillingPeriod.Annual
+	| typeof BillingPeriod.OneYearGift
```

Comment:
I don't think these should be in RecurringBillingPeriod because they are not recurring, they're a one time charge

Reply:
Yep makes sense, have replaced type `RecurringBillingPeriod` entirely with `BillingPeriod` now as we now include Gifting in this type.

Reply:
@rupertbates : Have now refactored without `BillingPeriod` changes.